ui-react/packages/atlasmap/src/Atlasmap/useAtlasmapDialogs.tsx

```diff
@@ -197,6 +224,63 @@ export function useAtlasmapDialogs({
     });
   }, [exportAtlasFile, openExportDialog]);
 
+  const onCreateNamespace = useCallback(
+    (docName: string) => {
+      title.current = docName;
+      textVal1.current = "";
+      textVal2.current = "";
+      textVal3.current = "";
+      booleanVal.current = false;
+      openCreateNamespaceDialog(
+        (
+          docName: string,
+          alias: string,
+          uri: string,
+          locationUri: string,
+          isTarget: boolean,
+        ) => {
+          createNamespace(docName, alias, uri, locationUri, isTarget);
+        },
+      );
+    },
+    [createNamespace, openCreateNamespaceDialog],
+  );
+
+  const onEditNamespace = useCallback(
+    (
+      docName: string,
+      alias: string,
+      uri: string,
+      locationUri: string,
+      isTarget: boolean,
+    ) => {
+      title.current = docName;
+      textVal1.current = alias;
+      textVal2.current = uri;
+      textVal3.current = locationUri;
+      booleanVal.current = isTarget;
+      openEditNamespaceDialog(
+        (
+          docName: string,
+          alias: string,
+          uri: string,
+          locationUri: string,
+          isTarget: boolean,
+        ) => {
+          editNamespace(docName, alias, uri, locationUri, isTarget);
+        },
+      );
+    },
+    [editNamespace, openEditNamespaceDialog],
+  );
+
+  const onDeleteNamespace = useCallback(
+    (docName: string, alias: string) => {
```

Comment:
Same. Also there is no need to wrap this in useCallback since the function doesn’t change between renders